ws-slave.js

```diff
@@ -117,52 +109,17 @@ if(defaultScanUuid) {
 
 console.log('noble - ws slave - server mode');
 
-myScanServer.on('connection', (socket) => {
-  console.log("scan server received incoming socket");
-  g_scannerSocket = socket;
-  socket.on('message', (msg) => {
-    const data = JSON.parse(msg);
-
-    if(data.evt === 'discover') {
-      // the scanner process has found a thing!  let's see which of our contexts would be interested in it.
-      const periph = data.data;
-      //console.log("scanproc told us about ", periph.advertisement.localName);
-      noble.onReviveStoredPeripheral(periph.uuid, periph.address, periph.addressType, periph.connectable, periph.advertisement, periph.rssi);
-    }
-
-  });
-  socket.on('close', (msg) => {
-    g_scannerSocket = null;
-  })
-});
-myScanServer.on('error', (err) => {
-  console.log("scan server had error", err);
-})
-let runArgs = process.argv.slice(1);
-runArgs = runArgs.map((arg) => arg.replace("ws-slave", "scanner"));
-console.log("making new scan process with ", runArgs);
-permaScanner = child_process.spawn("node", runArgs);
-permaScanner.on('close', (code) => {
-  console.log("scanner process closed ", code);
-})
-permaScanner.on('exit', (code) => {
-  console.log("scanner process exit ", code);
-  exitWithCode(1);
-})
-permaScanner.stdout.on('data', (chunk) => {
-  console.log("SCANPROC STDOUT: " + chunk.toString());
-})
-permaScanner.stderr.on('data', (chunk) => {
-  console.error("SCANPROC STDERR: " + chunk.toString());
-})
-
-
-
-
 const controlPort = 0xb1f;
 const myControlSocket = new WebSocket(`ws://localhost:${controlPort}`);
 myControlSocket.on('message', (msg) => {
-  
+  switch(msg) {
+  case 'enable-real-scanning':
+      console.log("##### got enable-real-scanning request");
```

Comment:
I guess since you've managed to make it always "real scanning" mode now, we can get rid of this stuff entirely